src/ServicePulse.Host/vue/src/resources/Endpoint.ts

```diff
@@ -0,0 +1,96 @@
+export interface Endpoint {
+  name: string;
+  errorCount: number;
+  serviceControlId: string;
+  isScMonitoringDisconnected: boolean;
+  metrics: EndpointMetrics;
+  isStale: boolean;
+  endpointInstanceIds: string[];
+  disconnectedCount: number;
+  connectedCount: number;
+}
+
+export interface EndpointValues {
+  points: number[];
+  average: number;
+}
+
+export interface EndpointValuesWithTime extends EndpointValues {
+  timeAxisValues: Date[];
```

Comment:
I assume this does not come from the backend REST API? 
c# dates do not desecrialize into JS dates

Reply:
it's actually not used... I have kept the member in case it was meant to be used somewhere which has been lost in migration, but changed the type to string[]